app/controllers/location.js

```diff
@@ -0,0 +1,55 @@
+const https = require("follow-redirects").https;
+require("dotenv").config();
+
+const placeDetails = function() {
+    this.places = [];
+};
+
+//Declare loction information
+const googleApiKey = process.env.GOOGLE_API;
```

Comment:
Please add the .env property and value in slack so I can add it to Heroku

Reply:
gedaan, zie slack 😄